cloud/deployments.md

```diff
@@ -5,4 +5,81 @@ nav_order: 1
 parent: GlareDB Cloud
 ---
 
-## Deployments
+# Deployments
+
+Deployments represent the persistent state of a GlareDB database.
```

Comment:
What does this mean?

Reply:
Maybe something that explains the difference between a deployment and a database would be useful.

Reply:
And also, something about when one would want to create a new deployment.

Reply:
The biggest piece of confusion is that we call it a Deployment, to be totally honest, we just haven't figured out what's better.

Our cloud is fully severless and multi-tenant.

This means that you don't actually deploy anything, you're just connected to shared compute. Storage is in an organization-scoped bucket in GCS in our cloud, and so really a deployment just means you have a bucket of storage, accessed by serverless compute.

The words were originally chosen here because running GlareDB locally has not persistence by default, or you can store things on disk.

Cloud is storing everything centrally in cloud.

So we could rework this to make it less confusing to explain:

- Cloud is serverless, with persistent storage
- You can connect to cloud locally, in notebooks, in applications and scripts, and access your data wherever you like

As for creating multiple deployments - it's a paid feature only, so we can omit that for now, I think and add it later

Reply:
https://github.com/GlareDB/glaredb.github.io/pull/241/commits/d15ab6456825ad3fc3b60385e8038de247bb2df7